package.json

```diff
@@ -25,11 +25,11 @@
     "build-dev": "npm run build-main-dev && npm run build-renderer-dev",
     "start": "cross-env NODE_ENV=production electron ./app/",
     "start-hot": "cross-env HOT=1 NODE_ENV=development electron -r babel-register -r babel-polyfill ./app/main.development",
-    "postinstall": "concurrently \"electron-builder install-app-deps\" \"node node_modules/fbjs-scripts/node/check-dev-engines.js package.json\" && electron-rebuild --force --module-dir app",
+    "postinstall": "concurrently \"install-app-deps\" \"node node_modules/fbjs-scripts/node/check-dev-engines.js package.json\" && electron-rebuild --force --module-dir app",
     "dev": "npm run hot-server -- --start-hot",
     "prepare-qa-build": "node -r babel-register scripts/prepare-qa-build.js",
     "package": "npm run build && build -p onTagOrDraft",
-    "package-dev": "npm run build-dev && build -p onTagOrDraft --config.artifactName=\"${productName}-${version}-${arch}-${os}-dev.${ext}\" --config.nsis.artifactName=\"${productName}-Setup-${version}-dev.${ext}\" --extraMetadata.name=\"tidepool-uploader-dev\"",
+    "package-dev": "npm run build-dev && build -p onTagOrDraft --config.artifactName='${productName}-${version}-dev.${ext}' --config.nsis.artifactName='${productName}-Setup-${version}-dev.${ext}' --extraMetadata.name='tidepool-uploader-dev'",
```

Comment:
@gniezen - This was me simply reverting to what previously worked. The conflict was a combination of going back to `dmg` from `pkg` (because the artifact gets a proper extension in the interim build step and has a double period `..` in the file name). Technically this isn't great, since Windows will leave in the single-quotes as literals, but it doesn't die when trying to reference the temp files the way that Mac will if we switch out the escaping scheme. I'm only mostly sure that there's no way to properly escape the `$` character as a command line option value inside a JSON file that will be properly cross-platform. These options could be set in a separate config file by moving the `build` key out of `package.json` into two new files, one for the production build and one for the dev build, but the files would be largely duplicates of one another (except for the addition of these keys). I try to stick to "don't repeat yourself", but wanted your opinion concerning moving this hack to two config files instead (which will hopefully help us in efforts to upgrade later).

Reply:
Maybe just leave it as is for now and then we can move to a two-file config when we upgrade later?